sopel/plugin.py

```diff
@@ -679,15 +681,18 @@ def action_command(*command_list: str) -> Callable:
 
     .. note::
 
-        You can use a regular expression for the command name(s), but this is
-        **not recommended** since version 7.1. For backward compatibility,
-        this behavior will be kept until version 8.0.
+        The command name will be escaped to be used in a regex command. As such
+        it is not possible to use something like ``/me command\\d+`` to catch
+        something like ``/me command1`` or ``/me command2``.
+
+        You have several options at your disposal to replace a regex in the
+        command name:
 
-        Regex patterns are confusing for your users; please don't use them in
-        command names!
+        * use a command alias,
+        * parse the arguments with your own regex within your plugin callable,
+        * use a :func:`rule`,
 
-        If you need to use a regex pattern, please use the :func:`rule`
-        decorator instead, with the :func:`ctcp` decorator::
+        the rule must be used with the :func:`ctcp` decorator::
```

Comment:
```suggestion
        The :func:`rule` must be used with the :func:`ctcp` decorator::
```

(Blame GH for forcing me to split this from the suggestion above due to deleted lines in between.)